README.md

```diff
@@ -97,6 +97,7 @@ $ node example.js | pino
 * `genReqId`: you can pass a function which gets used to generate a request id. The first argument is the request itself. As fallback `pino-http` is just using an integer. This default might not be the desired behavior if you're running multiple instances of the app
 * `useLevel`: the logger level `pino-http` is using to log out the response. default: `info`
 * `customLogLevel`: set to a `function (res, err) => { /* returns level name string */ }`. This function will be invoked to determine the level at which the log should be issued. This option is mutually exclusive with the `useLevel` option. The first argument is the HTTP response. The second argument is an error object if an error has occurred in the request.
+* `disableAutoLogging`: set to `true` to disable the automatic "request completed" and "request errored" logging. Defaults to `false`.
```

Comment:
I prefer positive flags: how about “autoLogging: true”?

Reply:
Sure, I can switch it - I just didn't want to change the default behavior. This would be a breaking change for anyone who expects the automatic logging since they would have to add the flag to their options object. Thoughts?

Reply:
Could we not just default the setting as true?

Reply:
My brain wasn't working for a moment.. I will make the changes later!